app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,159 @@ class MainActivity : ComponentActivity() {
+                    )
+                    .padding(all = 8.dp)
+                    .background(Color.Red)
+            )
+
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .clip(CircleShape)
+                    .background(Color.Blue)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 20.dp)
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(size = 80.dp)
+            )
+
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(size = 80.dp)
+                    .clip(CircleShape)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 10.dp)
+        ) {
+            Column (
+                modifier = modifier
+                    .size(240.dp,140.dp)
+                    .border(
+                        1.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 10.dp)
+                    )
+            ){
+                Column(
+                    modifier = Modifier.padding(16.dp),
+                            verticalArrangement = Arrangement.spacedBy(3.dp)
+                ) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.End
+                    ){
+                        Box{
+                            Image(
+                                painter = painterResource(id = R.drawable.image2),
+                                contentDescription = "쿠잇 이미지",
+                                modifier = Modifier
+                                    .size(size = 40.dp)
+                                    .clip(RoundedCornerShape(size = 10.dp))
+                            )
+                        }
+                    }
+
+                    Column(
+                        verticalArrangement = Arrangement.spacedBy(3.dp)
+                    ) {
+                        Divider(color = Color.Black, thickness = 0.5.dp)
+
+                        Column(
+                            verticalArrangement = Arrangement.spacedBy(1.dp)
+                        ) {
+                            Text("이름 : 이시헌", fontSize = 12.sp)
+                            Text("학번 : 202411348", fontSize = 10.sp)
+                            Text("학과 : 컴퓨터공학과", fontSize = 10.sp)
+                            Text("생년월일 : 2004/10/11", fontSize = 10.sp)
+                        }
```

Comment:
이렇게 작성하시면 위쪽에 작성했던 Column의 SpacedBy(3.dp) 안의 아이템은 이 Column과 Divider 두 아이템만 존재하는 걸로 되서 3.dp라는 패딩값이 텍스트간의 간격에 적용되지 않고 텍스트 간의 간격은 1.dp로 적용됩니다 ! 텍스트 간의 간격이 조금씩 다른 거 같아서 5.dp로 패딩을 주시고 이시헌이라는 텍스트 컴포넌트 하단에 추가적으로 패딩값을 주는 방식으로 작성하면 좋을 거 같습니다 ~
```suggestion
                       Column(
                            verticalArrangement = Arrangement.spacedBy(5.dp)
                        ) {
                            Text("이름 : 이시헌", fontSize = 12.sp, modifier = modifier.padding(bottom =4.dp ))
                            Text("학번 : 202411348", fontSize = 10.sp)
                            Text("학과 : 컴퓨터공학과", fontSize = 10.sp)
                            Text("생년월일 : 2004/10/11", fontSize = 10.sp)
                        }
```